openapi.yaml

```diff
@@ -140,7 +177,37 @@ components:
         - asyncapi
       properties:
         asyncapi:
-          $ref: '#/components/schemas/AsyncAPIDocument'
+          $ref: "#/components/schemas/AsyncAPIDocument"
+    BundleDocument:
+      type: object
+      required:
+        - asyncapis
+      optional:
+        - base  
+      properties:
+        asyncapis:
+          type: array
+          items:
+            $ref: "#/components/schemas/AsyncAPIDocument"
+        base:
+          type: string
+          description: Base directory to be used for bundling.
+          default: "."    
+    Bundle:
+      type: object
+      required:
+        - asyncapis
+      optional:
+        - base  
+      properties:
+        asyncapis:
+          type: array
+          items:
+            $ref: "#/components/schemas/AsyncAPIDocument"
+        base:
+          type: string
+          description: Base directory to be used for bundling.
+          default: "."    
```

Comment:
```suggestion
    Bundle:
      type: [object, string]
```

Reply:
You exactly return from that `/bundle` endpoint the object (JSON) or string (YAML) :)